web/main.ts

```diff
@@ -45,6 +48,24 @@ function saveBoundsSoon() {
   }, 1000);
 }
 
+function loadFrontend() {
+  numberRetries--;
+  if (electronStore.get('development').embedded || numberRetries < 0) {
+    if(numberRetries < 0) {
+      const alertOptions: MessageBoxOptions = {
+        type: 'warning',
+        buttons: ['OK'],
```

Comment:
Can we make this two buttons, `Retry` and `Cancel`

Where retry would attempt to reload the proxy URL (essentially running this method again)
And Cancel would reset to the internal assets.

Reply:
✅